content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -27,105 +26,61 @@ Install [`kubectl`](/docs/tasks/tools/#kubectl).
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-
 <!-- steps -->
 
 ## Re-encrypt Kubernetes secrets using storage version migration
-- To begin with, [configure KMS provider](/docs/tasks/administer-cluster/kms-provider/) 
+
+- To begin with, [configure KMS provider](/docs/tasks/administer-cluster/kms-provider/)
   to encrypt data at rest in etcd using following encryption configuration.
-  ```yaml
-  kind: EncryptionConfiguration
-  apiVersion: apiserver.config.k8s.io/v1
-  resources:
-  - resources:
-    - secrets
-    providers:
-    - aescbc:
-      keys:
-      - name: key1
-        secret: c2VjcmV0IGlzIHNlY3VyZQ==
-  ```
+
+  {{% code language="yaml" file="storage/encryption-configuration-key1.yaml" %}}
```

Comment:
I'm not convinced this warrants an example, for two reasons:

1. This is not a regular manifest for users to try out. It is a server side configuration file which happens to be formatted in YAML. It is only meant for cluster admins to understand how to configure encryption at rest.
2. There are confidential data in the YAML, and that data is only for demo's purpose. We are not supposed to encourage users to try this YAML as-is. However, rendering this into an example may confuse some users.

Reply:
For no.2, how about to replace with:

- `<your-secret-for-key1>`
- `<your-secret-for-key2>`

Reply:
Still, the so-called example, when extracted into a separate YAML, may be treated as something to try out as is. I'm concerned that some users may believe the YAML can be reused in their environments.

Reply:
Really don't know how. All changes to those two YAML have been reverted. Maybe someone can improve them in another PR, thanks.